sdks/python/apache_beam/yaml/yaml_transform.py

```diff
@@ -227,6 +227,8 @@ def get_pcollection(self, name):
       outputs = self.get_outputs(transform)
       if output in outputs:
         return outputs[output]
+      elif len(outputs) == 1 and outputs[next(iter(outputs))].tag == output:
```

Comment:
@robertwb I added this for cases where there is only 1 output from SchemaTransform (i.e. Write) because it seems to always get labeled as 'out' even when the `ErrorHandling` is specified